src/components/Dashboard/TodoList.vue

```diff
@@ -951,7 +952,7 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
         arMinDate: header.arMinDate, // COOP only
         arMaxDate: header.arMaxDate, // COOP only
         status: header.status || FilingStatus.NEW,
-        enabled: task.enabled,
+        enabled: task.enabled && !this.isDisableNonBenCorps,
```

Comment:
This is the method that loads the todo item for an Annual Report.

This code change sets `enabled` to False if the FF isn't set.

Screenshot when FF is True (ie, business is allowed).
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/f43a8fe4-59b8-45e0-a6dd-871b4525fde9)

Screenshot when FF is False (ie, business is not allowed).
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/cd12a3ac-0da7-402a-91f2-d4c2457820c4)